test/e2e/tests/metrics/traces.spec.ts

```diff
@@ -0,0 +1,129 @@
+import { MockttpServer } from 'mockttp';
+import FixtureBuilder from '../../fixture-builder';
+import {
+  defaultGanacheOptions,
+  unlockWallet,
+  withFixtures,
+} from '../../helpers';
+import {
+  expectMockRequest,
+  expectNoMockRequest,
+} from '../../helpers/mock-server';
+
+async function mockSentryCustomTrace(mockServer: MockttpServer) {
+  return [
+    await mockServer
+      .forPost(/sentry/u)
+      .withBodyIncluding('"transaction":"UI Startup"')
+      .thenCallback(() => {
+        return {
+          statusCode: 200,
+          json: {},
+        };
+      }),
+  ];
+}
+
+async function mockSentryAutomatedTrace(mockServer: MockttpServer) {
+  return [
+    await mockServer
+      .forPost(/sentry/u)
+      .withBodyIncluding('"transaction":"/home.html"')
+      .thenCallback(() => {
+        return {
+          statusCode: 200,
+          json: {},
+        };
+      }),
+  ];
+}
+
+describe('Traces', function () {
+  it('sends custom trace when opening UI if metrics enabled', async function () {
+    await withFixtures(
+      {
+        fixtures: new FixtureBuilder()
+          .withMetaMetricsController({
+            participateInMetaMetrics: true,
+          })
+          .build(),
+        ganacheOptions: defaultGanacheOptions,
+        title: this.test?.fullTitle(),
+        testSpecificMock: mockSentryCustomTrace,
+        manifestFlags: {
+          doNotForceSentryForThisTest: true,
+        },
```

Comment:
I commented out all of these
```
manifestFlags: {
 doNotForceSentryForThisTest: true,
},
```
and it still worked, is this expected?  I think that flag is mostly for tests with forced intentional errors.

Reply:
My reasoning here was a few different things:

- As these tests are verifying Sentry usage itself, it felt odd to use the results for additional metrics.
- When I ran locally with a custom DSN, I got snapshot errors as the host didn't match, this avoids that as it ensures the fake DSN is used.
- There are also negative tests to ensure this doesn't work if metrics are disabled, but I believe `flags.circleci` forces metrics to be enabled unless this manifest flag is set.